components/button/index.js

```diff
@@ -47,12 +49,14 @@ class Button extends Component {
 			...additionalProps
 		} = this.props;
 		const classes = classnames( 'components-button', className, {
-			button: ( isPrimary || isLarge || isSmall ),
-			'button-primary': isPrimary,
-			'button-large': isLarge,
-			'button-small': isSmall,
+			'is-button': isDefault || isPrimary || isLarge || isSmall,
+			'is-default': isDefault || isLarge || isSmall,
+			'is-primary': isPrimary,
+			'is-large': isLarge,
+			'is-small': isSmall,
 			'is-toggled': isToggled,
 			'is-busy': isBusy,
+			'is-link': isLink,
```

Comment:
There are way too many button states here, but this matches Core, we can try to rethink some of them later.

Reply:
Would it make sense to replace all these booleans with a more enum-like API? e.g. `<Button style="link">`?

Reply:
I don't know maybe later, let's not change all the buttons today :)